src/reports/models/database/cassandra/cassandraConnector.js

```diff
@@ -43,31 +47,50 @@ function insertReport(testId, revisionId, reportId, jobId, testType, phase, star
     let params;
     const testNotes = notes || '';
     params = [testId, revisionId, reportId, jobId, testType, phase, startTime, testName, testDescription, testConfiguration, testNotes, lastUpdatedAt];
+    insertLastReport(testId, revisionId, reportId, jobId, testType, phase, startTime, testName, testDescription, testConfiguration, notes, lastUpdatedAt);
```

Comment:
insertLastReport is async action which we are not waiting. no one will catch the error if error occurred, so insertLastReport should catch the error by itself and log a warning if error occurred.

Reply:
executeQuery log error in case we had an error in the query to Cassandra